### DISCUSSION TITLE
What are the exact dimensions of the Open Source Rover?

### DISCUSSION THREAD

**Discussion by ctech932** [nasa-jpl/open-source-rover#307] [Q&A] [answered]
Hi,
I am trying to find the exact dimensions of this Open Source Rover once it is completely built. I need to know the length, width and height (including the wheels).
I have seen that there is a specification chart in the Wiki section, in which the footprint is 60.96x30.48 [cm], but I do not understand clearly what the footprint is. I have also tried to use the Onshape model to measure the length and width (with a top view), and I find 750 mm for the length and 566 mm for the width. This does not correspond to the footprint given in the specification chart.
Could you please help me?
Thanks!

**Comment by JHPHELAN** [comment, marked as answer]
29.5" long, 22" wide, 16" high w/o the head (0.75m x 0.56m x 0.40m) as it sits on my desk.

**Reply by ctech932** [reply]
Thank you very much for those measurements
